site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -3860,7 +3860,12 @@ private void addAccess(Context context, Map<String, AdditionalRole> access) {
 	 */
 	protected Map<String, AdditionalRole> loadAdditionalRoles() {
 		Map<String, AdditionalRole> additionalRoles = new HashMap<String, AdditionalRole>();
-		for (String roleId : authzGroupService.getAdditionalRoles()) {			
+		for (String roleId : authzGroupService.getAdditionalRoles()) {
```

Comment:
Hi @buckett, so do you want me to move the property check to the Kernel and do not make any change over SiteAction? If yes i can make a second commit and new PR.